fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -57,6 +57,17 @@
       </xsl:otherwise>
         </xsl:choose>
     </field>
+
+    <-- JIRA TRAC-876 Define utk_mods_etd_author_orcid in Solr -->
+
+    <xsl:if test="mods:name[@type='orcid']">
```

Comment:
This isn't working.  One problem here is that you're reading the wrong attribute value.  You should be reading mods:name[@authority='orcid'].

Reply:
Your conditional needs to start with this:

<xsl:if test="/mods:mods/mods:name[@authority='orcid']">

Reply:
Revised test statement